model/license.go

```diff
@@ -224,6 +228,14 @@ func (f *Features) SetDefaults() {
 	if f.Cloud == nil {
 		f.Cloud = NewBool(false)
 	}
+
+	if f.SharedChannels == nil {
+		f.SharedChannels = NewBool(false)
+	}
+
+	if f.RemoteClusterService == nil {
+		f.RemoteClusterService = f.SharedChannels
```

Comment:
My understanding is that RemoteClusterService should not be configured without SharedChannels being configured. I will refer to @wiggin77 for asserting my assumption for this, in the meanwhile will guard against RemoteClusterService being nil.

Reply:
Now that I am thinking it more, `SharedChannels` cannot be nil since we set a default value to it before we set the `RemoteClusterService` default. So we should be fine on this.

Reply:
Good point! Complete overlook from my side 👍

Reply:
>  My understanding is that RemoteClusterService should not be configured without SharedChannels being configured. 

@nronas just for clarity, the Shared Channels service relies on Remote Cluster service for comms, therefore if Shared Channels is enabled, Remote Clusters must be as well.  Post MVP we'll create separate UI for enabling Remote Clusters independent of Shared Channels since other things will be shared/communicated between clusters.